v2/cmd/wails/internal/commands/dev/watcher.go

```diff
@@ -0,0 +1,67 @@
+package dev
+
+import (
+	"bufio"
+	"os"
+	"path/filepath"
+
+	"github.com/wailsapp/wails/v2/internal/fs"
+
+	"github.com/fsnotify/fsnotify"
+	gitignore "github.com/sabhiram/go-gitignore"
+	"github.com/samber/lo"
+)
+
+type Watcher interface {
```

Comment:
Seems like we don't need the Watcher interface at the moment. Should we remove it?

Reply:
I'll keep it in for now as I really do want to use FSEvents for Mac